templates/app-ts/__taprc

```diff
@@ -0,0 +1,3 @@
+test-env: [
+  TS_NODE_PROJECT=./test/tsconfig.test.json
```

Comment:
I like using this! Didn't know that it was possible to add env vars to `.taprc`! Well done.

Reply:
Recent discovery for me too. :partying_face: